src/theme/sass-partials/_sidebar.scss

```diff
@@ -49,36 +43,6 @@
   }
 }
 
-.split-pane-visible > .split-pane-side.mes-sideMenu {
-  z-index: 1;
-  border: none;
-  transition-timing-function: ease-out;
-  transition: 0.5s;
-  .toolbar-title img {
-    bottom: 7px;
-    position: absolute;
-    height: 35px;
-    left: 10px;
-  }
-  .toolbar-background {
-    background: map-get($colors, lighterDark);
-  }
-  .content {
-    background: map-get($colors, darkerGrey);
-  }
-  .footer {
-    padding-left: 20px;
-    font-size: 18px;
-    background: map-get($colors, lighterGrey);
-    height: 57px;
-    color: map-get($colors, textGrey);
-    line-height: 55px;
-    span {
-      margin-left: 20px;
-    }
-  }
-}
-
```

Comment:
Can we remove this file completely, now we no longer have the sidebar?  I think the footer styles could definitely go.